README.md

```diff
@@ -35,24 +35,26 @@ The structure of WD Firmware package allows quick and easy integration for more
 
 Coming soon: more FW features and more Platform supports 
 
-The following README file describes how to setup a build environment for the WD RISC-V firmware with HiFive1 board; It will build an example application - FreeRTOS demo and describe how to download and debug it. The flow was verified with VMWare player v. 15 hosting Debian 9.6. This demo is based on the work done here https://github.com/sifive/freedom-e-sdk/tree/FreeRTOS/software/FreeRTOSv9.0.0
+The following README file describes how to setup a build environment for the WD RISC-V firmware with HiFive1 board; It will build an example application - FreeRTOS demo and describe how to download and debug it. The flow was verified with VMWare player v. 15 hosting Debian 9.6. This demo was based on SiFive code which was ported official FreeRTOS maintement.
 
 ### Source tree structure 
 ```javascript
 WD-Firmware
-     |-board                                <-- supported boards
-          |-hifive-1                        
-          |-hifive-unleashed
-     |-common                               <-- common source
-     |-examples                             <-- examples source 
-          |-build                           <-- examples build scripts
-          |-ex-freertos                     <-- FreeRTOS example
-     |-psp                                  <-- psp functionality
-     |-rtos
-          |-rtosal                          <-- rtos abstraction layer
-          |-rtos_core                       <-- specific rtos source code
-                |-freertos
-                |-....
+     ├── board                                <-- supported boards
+          ├── hifive-1                        
+          ├── ihfive-unleashed (not supported yet)
+          ├── nexys_a7 (Support for SweRV v1)
+     ├── common                               <-- common source
+     ├── demos                                <-- demos source 
+          ├── build                           <-- examples build scripts
+          ├── demo_rtosal.c                   <-- Abstruction Layer (AL) demo on FreeRTOS
+          ├── main.c                          <-- The main of all demos

```

Comment:
there are also the "demo_platform_al.h+c files, don't you want to mention them here too?

Reply:
nope. no need it is just tree layout